src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveImm2UnaryOpTestTemplate.template

```diff
@@ -0,0 +1,355 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/******************************************************************************
+ * This file is auto-generated from a template file by the GenerateTests.csx  *
+ * script in tests\src\JIT\HardwareIntrinsics\X86\Shared. In order to make    *
```

Comment:
```suggestion
 * script in tests\src\JIT\HardwareIntrinsics\Arm\Shared. In order to make    *
```

Reply:
> script in tests\src\JIT\HardwareIntrinsics\X86\Shared

Also did all the other files in the same directory.